generators/dockertools/resources/go/services.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Go won't support these 2 services, right? Is this just a placeholder file? If it's not needed, we should delete it.